{
  "Faults": [
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-03T17:33:00.000+0000",
      "latitude": 51.60886256364756,
      "estimatedRestoration": "2023-04-05T19:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT1688'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT1688",
      "reference": "KT1688",
      "etaUpdate": true,
      "affectedCustomerCount": 1,
      "name": "UB9 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "UB9 6JY"
      ],
      "location": {
        "coordinates": [
          -0.47871135171246504,
          51.60886256364756
        ],
        "type": "Point"
      },
      "UUID": "ca3d3d06-6671-40dc-b17c-23852d0a3952",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -0.47871135171246504,
      "resolved": false
    },
    {
      "latitude": 51.50646418387752,
      "type": "HV",
      "reference": "KT2009",
      "etaUpdate": true,
      "affectedCustomerCount": 16,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "99587b7a-6169-458b-a15d-6e0d794e5f62",
      "minLongitude": -2.3332,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.2928417982762697,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-04T08:56:00.000+0000",
      "estimatedRestoration": "2023-04-04T22:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT2009'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT2009",
      "maxLongitude": -2.2709,
      "minLatitude": 51.4938,
      "name": "SN14 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GL9 1HR",
        "SN14 7JL",
        "SN14 7PA",
        "SN14 7PB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.3332,
              51.5224
            ],
            [
              -2.2709,
              51.5224
            ],
            [
              -2.2709,
              51.4938
            ],
            [
              -2.3332,
              51.4938
            ],
            [
              -2.3332,
              51.5224
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5224,
      "etrUpdate": true
    },
    {
      "latitude": 51.20832877974389,
      "type": "LV",
      "reference": "KT1967",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-04T09:00:00.000+0000",
      "affectedCustomerCount": 1,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "e57b34ba-f439-4660-aee7-b21a007fb826",
      "minLongitude": -1.4803,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.4799574790352379,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-04T08:22:00.000+0000",
      "estimatedRestoration": "2023-04-05T01:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT1967'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT1967",
      "maxLongitude": -1.4796,
      "minLatitude": 51.208,
      "name": "SP10 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SP10 1LT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.4803,
              51.2087
            ],
            [
              -1.4796,
              51.2087
            ],
            [
              -1.4796,
              51.208
            ],
            [
              -1.4803,
              51.208
            ],
            [
              -1.4803,
              51.2087
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-04T09:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2087,
      "etrUpdate": true
    },
    {
      "latitude": 51.71859096271214,
      "type": "HV",
      "reference": "KT2279",
      "etaUpdate": true,
      "affectedCustomerCount": 26,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "f932f8d3-9ba6-4b54-9edb-d79e5665100e",
      "minLongitude": -1.166,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.1335713625067976,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-04T12:02:00.000+0000",
      "estimatedRestoration": "2023-04-04T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT2279'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT2279",
      "maxLongitude": -0.932,
      "minLatitude": 51.4441,
      "name": "OX33 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "OX33 1NS",
        "OX33 1NT",
        "OX33 1NU",
        "OX33 1NX",
        "OX33 1NZ",
        "OX33 1TR",
        "RG6 1LP"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.166,
              51.7499
            ],
            [
              -0.932,
              51.7499
            ],
            [
              -0.932,
              51.4441
            ],
            [
              -1.166,
              51.4441
            ],
            [
              -1.166,
              51.7499
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-04T16:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.7499,
      "etrUpdate": true
    },
    {
      "latitude": 50.73244831481908,
      "type": "LV",
      "reference": "KS9814",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-01T16:30:00.000+0000",
      "affectedCustomerCount": 17,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "d00feb1d-ab39-4651-89df-f332134bf303",
      "minLongitude": -1.5924,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.5790725894630877,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-04T12:30:00.000+0000",
      "estimatedRestoration": "2023-04-04T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KS9814'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KS9814",
      "maxLongitude": -1.5403,
      "minLatitude": 50.721,
      "name": "SO41 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO41 0DA",
        "SO41 0PG",
        "SO41 0QD",
        "SO41 0QF",
        "SO41 0QG",
        "SO41 0QR",
        "SO41 0QS",
        "SO41 9AF",
        "SO41 9AL"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.5924,
              50.759
            ],
            [
              -1.5403,
              50.759
            ],
            [
              -1.5403,
              50.721
            ],
            [
              -1.5924,
              50.721
            ],
            [
              -1.5924,
              50.759
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.759,
      "etrUpdate": true
    },
    {
      "latitude": 51.558208233176835,
      "type": "LV",
      "reference": "KT2460",
      "etaUpdate": true,
      "affectedCustomerCount": 13,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "79ddf4f4-4b86-4c28-b14e-b64d185f38fe",
      "minLongitude": -0.8737,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8673493902188023,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-04T14:40:00.000+0000",
      "estimatedRestoration": "2023-04-05T02:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT2460'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT2460",
      "maxLongitude": -0.8469,
      "minLatitude": 51.5454,
      "name": "RG9 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG9 3AF",
        "RG9 3AY",
        "RG9 3BB",
        "RG9 3BD",
        "RG9 3BE",
        "RG9 3BH",
        "RG9 3BJ",
        "RG9 3BN"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8737,
              51.562
            ],
            [
              -0.8469,
              51.562
            ],
            [
              -0.8469,
              51.5454
            ],
            [
              -0.8737,
              51.5454
            ],
            [
              -0.8737,
              51.562
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-04T16:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.562,
      "etrUpdate": true
    },
    {
      "latitude": 50.74548202781766,
      "type": "LV",
      "reference": "KT2550",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-04T18:15:00.000+0000",
      "affectedCustomerCount": 13,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "ce25c41e-0941-4e54-8fc0-6ad0eebfff8e",
      "minLongitude": -1.8098,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7896719639817822,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-04T17:54:00.000+0000",
      "estimatedRestoration": "2023-04-05T00:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT2550'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT2550",
      "maxLongitude": -1.7532,
      "minLatitude": 50.725,
      "name": "BH23 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BH23 2GZ",
        "BH23 2LX",
        "BH23 2LZ",
        "BH23 2NA",
        "BH23 2NB",
        "BH23 2NP",
        "BH23 2PT",
        "BH6 5JG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.8098,
              50.7495
            ],
            [
              -1.7532,
              50.7495
            ],
            [
              -1.7532,
              50.725
            ],
            [
              -1.8098,
              50.725
            ],
            [
              -1.8098,
              50.7495
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7495,
      "etrUpdate": true
    },
    {
      "latitude": 51.72101864865611,
      "type": "HV",
      "reference": "KT2597",
      "etaUpdate": true,
      "affectedCustomerCount": 121,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "c4c46982-a62a-45f0-8d10-62ff442330f6",
      "minLongitude": -1.9707,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7272337684563124,
      "resolved": false,
      "py/object": "Fault",
      "Messages": [
        {
          "py/object": "Message",
          "senderName": "james.secker@sse.com",
          "senderUserId": "502b84f7-adfc-4d7c-b7ee-84662532950c",
          "text": "Engineers are responding to a main network fault in the area. We aim to restore power by 23:30.",
          "UUID": "84e3c5b6-36d6-4815-b232-ce7c7ae7b8cf",
          "py/class": "http://opengrid.com/givr/1.0#//Message",
          "sentDateTime": "2023-04-04T19:46:29.598+0000"
        }
      ],
      "srtUpdate": true,
      "loggedAt": "2023-04-04T19:26:00.000+0000",
      "estimatedRestoration": "2023-04-04T22:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT2597'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT2597",
      "maxLongitude": -1.6879,
      "minLatitude": 51.6916,
      "name": "GL7 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GL7 1HW",
        "GL7 3BE",
        "GL7 3DY",
        "GL7 3DZ",
        "GL7 3EA",
        "GL7 3EB",
        "GL7 3GH",
        "GL7 3NU",
        "GL7 3NX",
        "GL7 3PD",
        "GL7 3PG",
        "GL7 3PH",
        "GL7 3PJ",
        "GL7 3PN",
        "GL7 3PQ",
        "GL7 3PR",
        "GL7 3QA",
        "GL7 3QD",
        "GL7 3QF",
        "GL7 4PZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.9707,
              51.7325
            ],
            [
              -1.6879,
              51.7325
            ],
            [
              -1.6879,
              51.6916
            ],
            [
              -1.9707,
              51.6916
            ],
            [
              -1.9707,
              51.7325
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.7325,
      "etrUpdate": true
    },
    {
      "latitude": 50.94525081307599,
      "type": "LV",
      "reference": "KT2601",
      "etaUpdate": true,
      "affectedCustomerCount": 48,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "e95b66f7-dd45-4064-8788-cc6727b55042",
      "minLongitude": -2.5264,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.5224247668560205,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-04T19:34:00.000+0000",
      "estimatedRestoration": "2023-04-05T04:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT2601'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT2601",
      "maxLongitude": -2.5186,
      "minLatitude": 50.9432,
      "name": "DT9 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DT9 3AU",
        "DT9 3FA",
        "DT9 3HH",
        "DT9 3HJ",
        "DT9 3HL",
        "DT9 3HP",
        "DT9 3HQ",
        "DT9 3HS",
        "DT9 3HW",
        "DT9 6BH"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.5264,
              50.9484
            ],
            [
              -2.5186,
              50.9484
            ],
            [
              -2.5186,
              50.9432
            ],
            [
              -2.5264,
              50.9432
            ],
            [
              -2.5264,
              50.9484
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-04T20:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9484,
      "etrUpdate": true
    },
    {
      "latitude": 56.46871209199621,
      "type": "LV",
      "reference": "KT2616",
      "etaUpdate": true,
      "affectedCustomerCount": 29,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "776d6656-5eb7-4d08-a2e5-b8e0f2323785",
      "minLongitude": -2.9629,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.960317552265917,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-04T20:06:00.000+0000",
      "estimatedRestoration": "2023-04-04T22:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT2616'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT2616",
      "maxLongitude": -2.9578,
      "minLatitude": 56.4673,
      "name": "DD4 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DD4 6QT",
        "DD4 6QU",
        "DD4 6QZ",
        "DD4 6RA",
        "DD4 6RD",
        "DD4 6RE",
        "DD4 6RG",
        "DD4 6RS"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.9629,
              56.4701
            ],
            [
              -2.9578,
              56.4701
            ],
            [
              -2.9578,
              56.4673
            ],
            [
              -2.9629,
              56.4673
            ],
            [
              -2.9629,
              56.4701
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-04T21:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 56.4701,
      "etrUpdate": true
    }
  ]
}
